projects/step-frontend/src/lib/modules/timeseries/modules/chart/types/ts-chart-settings.ts

```diff
@@ -26,5 +26,6 @@ export interface XAxesSettings {
   show?: boolean; // default true
   time?: boolean; // default true
   label?: string;
+  incrs?: number[]; // specify multipliers of how the grid add more values when zooming.
```

Comment:
I'm not a fan of shortended words. Also I assumed it means increase but that doesn't really match the comment for me.

Reply:
@neogucky I also don't like shorhanded words. I used exactly the form they use in uPlot, just to not create confusion. To be honest, I don't know where it comes from. I can't even find a simple word to describe what it does, that's why I added the comment. I analyzed all the attributes and settings in the library, and I understood it from the documentation, not from its name. I could rename it to "gridDisplayMultipliers".